src/api/schemas/date.schema.js

```diff
@@ -0,0 +1,8 @@
+import Joi from 'joi';
+import JoiDate from "@joi/date";
+
+const extendedJoi = Joi.extend(JoiDate);
+
+export const dateSchema = extendedJoi.date().required()
+    .format('YYYY-MM-DD')
```

Comment:
How do you think about making it a constant ?

Reply:
I'll move it into constant if it appears more in one place. I propose leaving it as is. Could you, please, share your thoughts with me?

Reply:
yep, agree